static/js/components/applications/detail_sections_test.js

```diff
@@ -287,7 +287,14 @@ describe("application detail section component", () => {
       it(`should show correct link if ready === ${String(
         ready
       )}, fulfilled === ${String(fulfilled)}`, () => {
-        SETTINGS.novoed_login_url = "https://novoed.com"
+        SETTINGS.novoed_base_url = "https://novoed.com"
+        let novoedUrl = null
+        if (
+          SETTINGS.novoed_base_url &&
+          applicationDetail.bootcamp_run.novoed_course_stub
+        ) {
```

Comment:
The parameters for this test case should be updated, and a new param for the `novoed_course_stub` value added 
```typescript
    ;[
      [false, false, "", false, "not ready"],
      [true, false, "", false, "not fulfilled"],
      [true, true, "", false, "course stub is null"],
      [true, true, "some-stub", true, "fulfilled with a valid course stub"],
    ].forEach(([ready, fulfilled, novoEdStub, expLink, desc]) => {
      it(`${shouldIf(expLink)} show correct link if ${desc}`, () => {
        applicationDetail.bootcamp_run.novoed_course_stub = novoEdStub
        //...
       if (expLink) {
          assert.equal(link.prop("children"), expLinkText)
          assert.equal(link.prop("href"), `${SETTINGS.novoed_base_url}/#!/courses/${novoEdStub}/home`)
        }
```